glean-core/ios/Glean/Dispatchers.swift

```diff
@@ -43,39 +49,27 @@ class Dispatchers {
     // property
     private init() {}
 
-    /// This function launches a background `Operation`
+    /// This function launches a background `Operation` on a serially executed queue.
     ///
     /// - parameters:
     ///   * block: The block of code to execute, as a Closure that accepts no arugments and returns Void
```

Comment:
Now that I re-read this I think it's weird that we explain the closure. That's what the type system is for.